miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/RunRestController.java

```diff
@@ -113,18 +162,43 @@ public void setSecurityManager(SecurityManager securityManager) {
     return mapper.writeValueAsString(r);
   }
 
-  @RequestMapping(value = "{runAlias}/samplesheet", method = RequestMethod.GET)
-  public @ResponseBody String getSampleSheetForRun(@PathVariable String runAlias) throws IOException {
-    Run r = runService.getRunByAlias(runAlias);
+  @RequestMapping(value = "/{runId}/samplesheet", method = RequestMethod.GET)
+  public HttpEntity<String> getSampleSheetForRun(@PathVariable Long runId, HttpServletResponse response) throws IOException {
+    Run run = runService.get(runId);
+    return getSampleSheetForRun(run, SampleSheet.CASAVA_1_8, response);
```

Comment:
Will end up throwing NPE if the run doesn't exist, resulting in 500 status. Should instead return 404 status. Same with other similar methods in this class